CHANGELOG.md

```diff
@@ -7,7 +7,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 ## Unreleased
-
 ### Added
+- Added `MRR (Mean Reciprocal Rank)` to `metrics` ([#29](https://github.com/MobileTeleSystems/RecTools/pull/29))
+- Added `F1beta`, `MCC (Matthew correlation coefficient)` to `metrics` ([#32](https://github.com/MobileTeleSystems/RecTools/pull/32))
+- Added `LastNSplitter` to `model_selection` ([#33](https://github.com/MobileTeleSystems/RecTools/pull/32))
+- Added random `KFoldSplitter` to `model_selection` ([#31](https://github.com/MobileTeleSystems/RecTools/pull/31))
+- 
+
+### Changed
+- Bumped `implicit` version to 0.7.1 ([#45](https://github.com/MobileTeleSystems/RecTools/pull/45))
+- Bumped `poetry` version to 1.4.0 ([#43](https://github.com/MobileTeleSystems/RecTools/pull/43))
+- Updated dependencies in poetry lockfile ([#43](https://github.com/MobileTeleSystems/RecTools/pull/43))
+- Moved `nmslib` from main dependencies to extras ([#36](https://github.com/MobileTeleSystems/RecTools/pull/36))
+- Added base `Splitter` class to construct data splitters ([#31](https://github.com/MobileTeleSystems/RecTools/pull/31))
+- Updated notebooks in `examples` ([#44](https://github.com/MobileTeleSystems/RecTools/pull/44))
+
+### Fixed
+- fixed bugs with new version of `pytorch_lightning` ([#43](https://github.com/MobileTeleSystems/RecTools/pull/43))
```

Comment:
Points in other sections start with a capital letter, but fixes apparently do not deserve it :joy:

Reply:
Fixed the Fixes :)